addons/stock_landed_costs/product.py

```diff
@@ -16,7 +16,11 @@ class product_template(osv.osv):
 
     _columns = {
         'landed_cost_ok': fields.boolean('Landed Costs'),
-        'split_method': fields.selection(SPLIT_METHOD, 'Split Method'),
+        'split_method': fields.selection(SPLIT_METHOD, 'Split Method', help="Equal : Cost will be divided in equal parts per product.\n"\
+            "By Quantity : Cost will be divided according to product's quantity.\n"\
```

Comment:
Copy paste the tooltip from the pad. There are some grammar mistakes : 
     Equal : Cost will be equally divided.
     By Quantity : Cost will be divided according to the product's quantity.
     By Current cost : Cost will be divided according to the product's current cost.
     By Weight : Cost will be divided depending on its weight.
     By Volume : Cost will be divided depending on its volume.